src/components/TeamGeneratorPage/TeamsDisplay.js

```diff
@@ -15,6 +14,7 @@ const TeamsDisplay = ({ playerList }) => {
 
 	function clearButton() {
 		window.dispatchEvent(new Event('clear'));
```

Comment:
Why the hell do you need an event listener??
If you need a state management use MobX

Reply:
to manage states and logic from different components